src/fileHandling/EncryptedFileReader.java

```diff
@@ -0,0 +1,16 @@
+package fileHandling;
+
+import java.util.regex.Pattern;
+
+public class EncryptedFileReader implements ReadableFile
+{
+    Pattern pattern;
```

Comment:
Add access mofier to the object as you need it ( apply least privilege concept )